config/locales/zh.yml

```diff
@@ -616,15 +616,15 @@ doc_auth.info.stepping_up_html: Verify your identity again to access this servic
 doc_auth.info.tag: 建议
 doc_auth.info.upload_from_computer: 没有手机？从该电脑上传你身份证件的照片。
 doc_auth.info.upload_from_phone: 你不必重新登录，而且拍完照后可以转回到该电脑。你的手机必须带有相机和网络浏览器。
-doc_auth.info.verify_at_post_office_description: 如果你没有移动设备或者无法轻松拍身份证件照片，这一选项更好。
+doc_auth.info.verify_at_post_office_description: 如果你没有手机拍身份证件照片，这一选项更好。
 doc_auth.info.verify_at_post_office_description_selfie: 如果你没有手机可以拍照，请选择该选项
-doc_auth.info.verify_at_post_office_instruction: 你在网上输入身份证件信息，然后到一个参与邮件去亲身验证身份。
+doc_auth.info.verify_at_post_office_instruction: 你在网上输入身份证件信息，然后到一个参与本项目的邮局去亲身验证身份。
 doc_auth.info.verify_at_post_office_instruction_selfie: 你在网上输入身份证件信息，然后到一个参与本项目的邮局去亲身验证身份。
-doc_auth.info.verify_at_post_office_link_text: 对亲身验证获得更多了解
+doc_auth.info.verify_at_post_office_link_text: 对亲身验证获得更多了解。
 doc_auth.info.verify_identity: 我们会要求获得你的个人信息并通过与公共记录核对来验证你的身份。
 doc_auth.info.verify_online_description: 如果你没有移动设备或者无法轻松拍身份证件照片，这一选项更好。
 doc_auth.info.verify_online_description_selfie: 如果你有手机可以拍照，请选择该选项。
-doc_auth.info.verify_online_instruction: 你将拍自己身份证件的照片来完全在网上验证身份。大多数用户都能轻松完成这一流程。
+doc_auth.info.verify_online_instruction: 你将拍身份证件的照片来完全在网上验证身份。大多数用户都能轻松完成这样流程。
```

Comment:
@KeithNava  Double check doc_auth.info.verify_online_instruction. I can find the previous translation in the translation doc but not this one.